protocols/v2/noise-sv2/src/responder.rs

```diff
@@ -122,16 +122,16 @@ impl Responder {
 
     /// #### 4.5.1.2 Responder
     ///
-    /// 1. receives ephemeral public key message with ElligatorSwift encoding (64 bytes plaintext) 
-    /// 2. obtain the 32 bytes encoding of x-coordinates of remote ephemeral PubKey parses public 
+    /// 1. receives ephemeral public key message with ElligatorSwift encoding (64 bytes plaintext)
+    /// 2. obtain the 32 bytes encoding of x-coordinates of remote ephemeral PubKey parses public
     ///    key as `re.public_key`
     /// 3. calls `MixHash(re.public_key)`
```

Comment:
In https://github.com/Sjors/bitcoin/pull/28 I use the EllSwift encoding for MixHash. 

That's also how I defined it in the spec proposal, though that could be made more clear: https://github.com/stratum-mining/sv2-spec/pull/66